database/test/db-admin-rolepermissions.test.js

```diff
@@ -0,0 +1,23 @@
+const assert = require('node:assert')
+const { describe, it } = require('mocha')
+const { usherDb } = require('../layer/knex')
+const adminRolePermissions = require('../layer/admin-rolepermissions')
+
+describe('Admin role permissions view', () => {
+  describe('Test getRolePermissions', () => {
+    const invalidRoleKey = 0
+    it('Should return an array of permissions for the role', async function () {
```

Comment:
@k1ch  how come you didn't use the arrow syntax here? 
ie:

```javascript
async () => {
```

Reply:
We need the generic `function` to access `this` in the test context since we have `this.skip()` in line 12.